src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/View.cshtml

```diff
@@ -65,7 +66,7 @@
             {
```

Comment:
The Delivery Model should be hidden if it's `Regular`. Most delivery models are Regular, and where this is the case most users won't even know what it means.

Reply:
Fixed in lastest commit